modules/dxp/apps/search-experiences/search-experiences-web/src/main/resources/META-INF/resources/sxp_blueprint_admin/js/shared/PageToolbar.js

```diff
@@ -109,7 +109,7 @@ export default function PageToolbar({
 				aria-label={Liferay.Language.get('page-toolbar')}
 				light
 			>
-				<ClayLayout.ContainerFluid size={false}>
+				<ClayLayout.ContainerFluid size="xxxl">
```

Comment:
Should it be changed from {false} to "xxxl" on [374](https://github.com/liferay-platform-experience/liferay-portal/blob/776958a19890dc3674b21e21c6c571f269447b3f/modules/dxp/apps/search-experiences/search-experiences-web/src/main/resources/META-INF/resources/sxp_blueprint_admin/js/shared/PageToolbar.js#L374) too? I noticed the feature flag was here before.

Reply:
Thanks @SelenaAungst, I can't find where that's rendered, but I'm sure it should have been changed.